docs/examples.md

```diff
@@ -0,0 +1,184 @@
+# Examples
+
+Here you can find a bunch of simple examples for using fcct, with some explanations about what they do. The examples here are in no way comprehensive, for a full list of all the options present in ct check out the [configuration specification][spec].
+
+## Users and groups
+
+```yaml fedora-coreos-config
+variant: fcos
+version: 1.0.0
+passwd:
+  users:
+    - name: core
+      password_hash: "$6$43y3tkl..."
+      ssh_authorized_keys:
+        - key1
+```
+
+This example modifies the existing `core` user, giving it a known password hash (this will enable login via password), and setting its ssh key.
```

Comment:
The Ignition docs have the description of the example above the example, should we follow that pattern?